src/actions/signupActions.js

```diff
@@ -0,0 +1,88 @@
+import axios from 'axios';
```

Comment:
@gracelungu `axios` should be customized as we shall also be adding the user's token